### PR TITLE
Create grape integration documentation.

### PR DESCRIPTION
**Pull request by xn**
#### Purpose

To inform grape integrators what is necessary to meet the formatter's dependencies.
#### Changes
Added docs

### CODE REVIEW

File: docs/howto/grape_integration.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+The AMS grape formatter relies on the existence of `env['grape.request']` which is implemeted by `Grape::Middleware::Globals`. You can meet his dependency by calling it before mounting the endpoints.
```
Comment (bf4) [inline docs/howto/grape_integration.md]: Rename ams to ActiveModelSerializers and looks good

Comment (bf4) [inline docs/howto/grape_integration.md]: Missing back to guides link at top, see other pages for example

Also please link from integrations table

I know this is mentioned elsewhere

**Comment by bf4** [comment]
@xn I ended up making the changes myself. Merged

**Comment by xn** [comment]
Sorry, I think I still have the work on my hard drive. Life. Sorry. Thanks.

**Pull request merged**